apps/api/src/persistence/database/utilities/convertResourceCompositeIdentifierToArangoDocumentHandle.ts

```diff
@@ -0,0 +1,8 @@
+import { ResourceCompositeIdentifier } from 'apps/api/src/domain/models/types/entityCompositeIdentifier';
+import { getArangoCollectionIDFromResourceType } from '../getArangoCollectionIDFromResourceType';
+import { ArangoCollectionID } from '../types/ArangoCollectionId';
+
+type ArangoDocumentHandle = `${ArangoCollectionID}/${string}`;
```

Comment:
String interpolation? Is this dividing a number to a string?

Reply:
I should probably mention `template literal types` in the `pro-D` channel. This has nothing to do with division as this is strictly a type- a specific subtype of string that is defined by pattern matching on the expression.